components/brave_rewards/resources/ui/components/settingsPage.tsx

```diff
@@ -77,6 +79,25 @@ class SettingsPage extends React.Component<Props, {}> {
     }
   }
 
+  getAdsNotifications = () => {
+    let difference
+    const now = new Date().getTime()
+    const date = new Date().getDate()
+    const day = (1000 * 60 * 60 * 24)
+
+    // Payment date is hard coded as the fifth for now
+    if (date > 5) {
```

Comment:
this needs to be separate variable so that it's easy to change. Maybe in utils so that it can be used in ads box `NextContribution` component as well.

Reply:
@ryanml can you please resolve this issue please